tests/test_A0_initialization.py

```diff
@@ -93,14 +106,21 @@ def test_check_input_path_posix():
     assert folder == os.path.join(REPO_PATH, "inputs")
 
 
+OUTPUT_PATH = os.path.join(".", "tests", "other")
+
+
 class TestCommandLineInput:
 
     parser = initializing.create_parser()
 
-    def test_input_file(self):
-        parsed = self.parser.parse_args(["-i", "input_file"])
+    def test_input_json_file(self):
+        parsed = self.parser.parse_args(["-json", "input_file"])
         assert parsed.path_input_file == "input_file"
 
+    def test_input_csv_file(self):
```

Comment:
not sure what this does/tests

Reply:
Same as above

Reply:
Should we test the default values, too? If no input is given the code needs to work with the default values.

Reply:
Yes good point :)